CONTRIBUTING.md

```diff
@@ -1 +1,41 @@
-Please run `./mvnw formatter:format` before sending a pull request.
+# YAVI contribution guidelines
+
+Great to hear you would like to contribute to YAVI! 🥳👍🏽
+
+## How to contribute
+
+Before you contribute your changes please make sure you follow the guidelines in this file.
+
+* Please run `./mvnw formatter:format` to make sure your follow the code style guidelines.
+* Make sure you follow the [Commit messages](#commit-messages) guidelines.
+* Make sure you describe your changes inside the pull request, so we get a better overview why this change is needed.
+* Add few test cases (unit or integration tests) that back your changes.
+   
+## Source Code style
+
+In this section we will provide you with information on how to format your code and import the styles of this project.
+
+* You can find IDE settings to import at `src/etc/`.
+* This project uses a maven formatter plugin that will autoformat your code. You can expect that
+  calling `./mvnw formatter:format` is the correct code style. Therefore, please run it before commiting.
+
+## Commit messages
+The commit message should follow the following style:
```

Comment:
Can you add this section is copied from spring-projects?

Reply:
Done